igb/repl.go

```diff
@@ -38,172 +38,150 @@ const (
 	emojis = "😀😁😂🤣😃😄😅😆😉😊😋😎😍😘😗😙😚🙂🤗🤔😐😑😶🙄😏😮😪😴😌😛😜😝🤤🙃🤑😲😭😳🤧😇🤠🤡🤥🤓😈👿👹👺💀👻👽🤖💩😺😸😹😻😼😽"
 )
 
+// Igb holds internal states of Igb.
+type Igb struct {
+	sm        *fsm.FSM
+	rl        *readline.Instance
+	completer *readline.PrefixCompleter
+	line      string
+	cmds      []string
+	stack     int
```

Comment:
Actually I still don't understand what `stack` in igb means. I may confuse people since normally we think `stack` as a kind of data structure.